gem/styles/gem-rwanda/modules/_m-article-teasers.scss

```diff
@@ -108,4 +108,10 @@
       }
     }
   }
+  .base-teaser__title{
```

Comment:
@sewagodimo Please locate which UI that has the classes. 
And then check where this element picking up these styles rules.